### PR TITLE
Elcomsoft.PhoneBreaker version 10.12.38835.6971

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/102152)

### CODE REVIEW

File: manifests/e/ElcomsoftCo/ElcomsoftPhoneBreaker/10.12.38835.6971/ElcomsoftCo.ElcomsoftPhoneBreaker.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: ElcomsoftCo.ElcomsoftPhoneBreaker
```
Comment (Trenly) [inline manifests/e/ElcomsoftCo/ElcomsoftPhoneBreaker/10.12.38835.6971/ElcomsoftCo.ElcomsoftPhoneBreaker.installer.yaml:4]:
```suggestion
PackageIdentifier: ElcomsoftCo.PhoneBreaker
```

File: manifests/e/ElcomsoftCo/ElcomsoftPhoneBreaker/10.12.38835.6971/ElcomsoftCo.ElcomsoftPhoneBreaker.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: ElcomsoftCo.ElcomsoftPhoneBreaker
```
Comment (Trenly) [inline manifests/e/ElcomsoftCo/ElcomsoftPhoneBreaker/10.12.38835.6971/ElcomsoftCo.ElcomsoftPhoneBreaker.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: ElcomsoftCo.PhoneBreaker
```

File: manifests/e/ElcomsoftCo/ElcomsoftPhoneBreaker/10.12.38835.6971/ElcomsoftCo.ElcomsoftPhoneBreaker.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: ElcomsoftCo.ElcomsoftPhoneBreaker
```
Comment (Trenly) [inline manifests/e/ElcomsoftCo/ElcomsoftPhoneBreaker/10.12.38835.6971/ElcomsoftCo.ElcomsoftPhoneBreaker.yaml:4]:
```suggestion
PackageIdentifier: ElcomsoftCo.PhoneBreaker
```

File: manifests/e/ElcomsoftCo/ElcomsoftPhoneBreaker/10.12.38835.6971/ElcomsoftCo.ElcomsoftPhoneBreaker.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: ElcomsoftCo.ElcomsoftPhoneBreaker
+PackageVersion: 10.12.38835.6971
+PackageLocale: en-US
+Publisher: Elcomsoft Co. Ltd.
+PackageName: Elcomsoft Phone Breaker
+License: Trialwatr
+ShortDescription: Break Phones
```
Comment (Trenly) [inline manifests/e/ElcomsoftCo/ElcomsoftPhoneBreaker/10.12.38835.6971/ElcomsoftCo.ElcomsoftPhoneBreaker.locale.en-US.yaml:10]: Can you provide a better description?

File: manifests/e/ElcomsoftCo/ElcomsoftPhoneBreaker/10.12.38835.6971/ElcomsoftCo.ElcomsoftPhoneBreaker.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: ElcomsoftCo.ElcomsoftPhoneBreaker
+PackageVersion: 10.12.38835.6971
+PackageLocale: en-US
+Publisher: Elcomsoft Co. Ltd.
+PackageName: Elcomsoft Phone Breaker
+License: Trialwatr
```
Comment (Trenly) [inline manifests/e/ElcomsoftCo/ElcomsoftPhoneBreaker/10.12.38835.6971/ElcomsoftCo.ElcomsoftPhoneBreaker.locale.en-US.yaml:9]:
```suggestion
License: Trialware
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by stephengillie** [comment]
Automatic Validation failed with:
```
Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. actual 
'manifests\e\ElcomsoftCo\ElcomsoftPhoneBreaker\10.12.38835.6971', expected 
'manifests\e\ElcomsoftCo\PhoneBreaker\10.12.38835.6971' 
```

The path has an extra `Elcomsoft` before `PhoneBreaker`.

**Comment by upintheairsheep** [comment]
Anyone using a git client that supports moving multiple items, or renaming a folder

File: manifests/e/Elcomsoft/PhoneBreaker/10.12.38835.6971/Elcomsoft.PhoneBreaker.yaml [hunk 6]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: ElcomsoftCo.PhoneBreaker
```
Comment (stephengillie) [inline manifests/e/Elcomsoft/PhoneBreaker/10.12.38835.6971/Elcomsoft.PhoneBreaker.yaml:4]:
```suggestion
PackageIdentifier: Elcomsoft.PhoneBreaker
```

File: manifests/e/Elcomsoft/PhoneBreaker/10.12.38835.6971/Elcomsoft.PhoneBreaker.locale.en-US.yaml [hunk 7]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: ElcomsoftCo.PhoneBreaker
```
Comment (stephengillie) [inline manifests/e/Elcomsoft/PhoneBreaker/10.12.38835.6971/Elcomsoft.PhoneBreaker.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: Elcomsoft.PhoneBreaker
```

File: manifests/e/Elcomsoft/PhoneBreaker/10.12.38835.6971/Elcomsoft.PhoneBreaker.installer.yaml [hunk 8]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: ElcomsoftCo.PhoneBreaker
```
Comment (stephengillie) [inline manifests/e/Elcomsoft/PhoneBreaker/10.12.38835.6971/Elcomsoft.PhoneBreaker.installer.yaml:4]:
```suggestion
PackageIdentifier: Elcomsoft.PhoneBreaker
```

Comment (stephengillie) [reply manifests/e/Elcomsoft/PhoneBreaker/10.12.38835.6971/Elcomsoft.PhoneBreaker.installer.yaml:4]: This will match the other Elcomsoft packages.

**Comment by upintheairsheep** [comment]
> Hello @upintheairsheep,
> 
> I am sorry to report that the Sha256 Hash does not match the installer. This may be caused by using a vanity URL rather than a URL directly to the binary.
> 
> Please check the HASH and submit again. Sorry for the inconvenience.
> 
> Template: msftbot/validationError/installers/hashMismatch

It most likely updated while it was “gone”

File: manifests/e/Elcomsoft/PhoneBreaker/10.12.38835.6971/Elcomsoft.PhoneBreaker.installer.yaml [hunk 9]
```diff
@@ -0,0 +1,14 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Elcomsoft.PhoneBreaker
+PackageVersion: 10.12.38835.6971
+Installers:
+- InstallerUrl: https://www.elcomsoft.com/download/eppb_setup_en.msi
+  InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: wix
+  InstallerSha256: 15D6497BEFDC994FC05E3B357F6338179B5F6B56B607FE653981B8C997796D1E
```
Comment (stephengillie) [inline manifests/e/Elcomsoft/PhoneBreaker/10.12.38835.6971/Elcomsoft.PhoneBreaker.installer.yaml:11]:
```suggestion
  InstallerSha256: E036DD3DD3987600537CF4D6C79201982FAC75D4502D99E060A0021B47C7C287
```

**Comment by stephengillie** [comment]
This package has the wrong EXE in the shortcut. It has `EPB.exe` but running this does nothing, even directly. But `EPBMain.exe` runs the program.

File: manifests/e/Elcomsoft/PhoneBreaker/10.12.38835.6971/Elcomsoft.PhoneBreaker.locale.en-US.yaml [hunk 10]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Elcomsoft.PhoneBreaker
+PackageVersion: 10.12.38835.6971
+PackageLocale: en-US
+Publisher: Elcomsoft Co. Ltd.
+PackageName: Elcomsoft Phone Breaker
+License: Trialware
+ShortDescription: Break Phones
```
Comment (stephengillie) [inline manifests/e/Elcomsoft/PhoneBreaker/10.12.38835.6971/Elcomsoft.PhoneBreaker.locale.en-US.yaml:10]:
```suggestion
ShortDescription: Perform logical and over-the-air acquisition of iOS devices, break into encrypted backups, obtain and analyze backups, synchronized data and passwords from Apple iCloud.
```

File: manifests/e/ElcomsoftCo/ElcomsoftPhoneBreaker/10.12.38835.6971/ElcomsoftCo.ElcomsoftPhoneBreaker.locale.en-US.yaml (hunk 4 above)
Comment (stephengillie) [reply manifests/e/ElcomsoftCo/ElcomsoftPhoneBreaker/10.12.38835.6971/ElcomsoftCo.ElcomsoftPhoneBreaker.locale.en-US.yaml]: Resolving this as I've suggested a paragraph from a potential [PackageUrl](https://www.elcomsoft.com/eppb.html) below.

**Comment by stephengillie** [comment]
> This package has the wrong EXE in the shortcut. It has `EPB.exe` but running this does nothing, even directly. But `EPBMain.exe` runs the program.

Editing the shortcut fixes the issue, allowing the shortcut to launch the application. I'm not sure how to fix this, as I believe it's part of the installer.

**Comment by upintheairsheep** [comment]
> > This package has the wrong EXE in the shortcut. It has `EPB.exe` but running this does nothing, even directly. But `EPBMain.exe` runs the program.
> 
> Editing the shortcut fixes the issue, allowing the shortcut to launch the application. I'm not sure how to resolve this, as I believe it's part of the installer.

It’s most likely an elcomsoft problem

**Comment by stephengillie** [comment]
Is there another version of this package where the shortcut has been fixed? It's a poor user experience to have a shortcut that doesn't do anything after installing software, and no way to run it.

**Comment by upintheairsheep** [comment]
Contact elcomsoft

**Comment by upintheairsheep** [comment]
> Is there another version of this package where the shortcut has been fixed? It's a poor user experience to have a shortcut that doesn't do anything after installing software, and no way to run it.

This is an Elcomsoft problem, contact them to report the bug, merge this into there, and this is a dynamicly updating link.

**Comment by upintheairsheep** [comment]
> > Is there another version of this package where the shortcut has been fixed? It's a poor user experience to have a shortcut that doesn't do anything after installing software, and no way to run it.
> 
> This is an Elcomsoft problem, contact them to report the bug, merge this into there, and this is a dynamicly updating link.

Follow up.

**Comment by stephengillie** [comment]
This appears to be the expected behavior for this package.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**